charts/opentelemetry-operator/templates/clusterrole.yaml

```diff
@@ -105,6 +105,18 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
+      - ""
+    resources:
+      - nodes/proxy
+    verbs:
+      - get
+  - apiGroups:
+      - ""
+    resources:
+      - nodes/stats
+    verbs:
+      - get
```

Comment:
@jaronoff97 if it is the collector that needs the permissions (I believe this is kubeletstatsreceiver or k8sattributesprocessor), why does the operator need them?

Reply:
I think this comes from the operator's RBAC magic that allows it to create the necessary RBAC for the collector (getting this from [the issue](https://github.com/open-telemetry/opentelemetry-helm-charts/issues/1434)) I think as a result this permission should only be in the section [here](https://github.com/open-telemetry/opentelemetry-helm-charts/pull/1436/files#diff-306d86f5a6db61d4fd383f2e0a05720ed2b0124de0daade5276559f83142dc37R76) which contains the permissions for operator RBAC magics

Reply:
Shall I move it inside the createRbacPermissions block then?

Reply:
Any updates here?